client/stores/useAuth.ts

```diff
@@ -1,10 +1,3 @@
-// export interface User {
-//   id: string
-//   email: string
-//   username: string
-//   role: string
-// }
-
```

Comment:
We are not longer planning to store user information in the store??

Reply:
we will ; but the interface probably wont look like that anyway, so, i just removed the comment for now